### PR TITLE
Fix implementation of enable attribute

### PR DESCRIPTION
**Pull request by rohitthakur2590**
##### SUMMARY

Depends-On: https://github.com/ansible/ansible-zuul-jobs/pull/1796

##### ISSUE TYPE

- Bugfix Pull Request
 

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by KB-perByte — APPROVED** [review]

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

File: changelogs/fragments/fix_enabling.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - fix the implementation of disabling interface.
```
Comment (Qalthos) [inline changelogs/fragments/fix_enabling.yaml:3]:
```suggestion
  - junos_interfaces - fix the behavior of ``enabled`` option. (https://github.com/ansible-collections/junipernetworks.junos/pull/401)
```

**Review by Qalthos — CHANGES_REQUESTED** [review]

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

**Comment by komaldesai13** [comment]
recheck

**Comment by komaldesai13** [comment]
reacheck

**Comment by komaldesai13** [comment]
recheck

**Comment by komaldesai13** [comment]
recheck

**Comment by komaldesai13** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

**Pull request merged**